### ISSUE TITLE
hyper-parameters used in algorithm 2

### ISSUE THREAD

**Issue by Tsingularity** [vvictoryuki/FreeDoM#8]
Hi, thanks for the great work!

Just curious what're the hyper-parameters used in the algorithm 2 (image below)? For example, how to set the learning rate and repeat time for each time step? I passed the paper but didn't find any detailed about this. Could you please share them? Thanks!

<img width="953" alt="image" src="https://github.com/vvictoryuki/FreeDoM/assets/13026942/835580b2-8d4e-4ecd-9f4e-116152a3e9ed">

**Comment by vvictoryuki** [comment]
@Tsingularity Thank you for recognizing our work! The hyperparameters in the algorithm are crucial for achieving good results. Regarding the repeat time, in our experiments, a higher number of repetitions usually does not harm the generation quality. Therefore, it is possible to search for a value that balances the computational cost and the quality of the generated results. However, the specific repeat time may vary depending on the dataset and conditions. As for setting the learning rate, it is a key factor in obtaining stable and high-quality generation results. The empirical strategies for its setting are relatively complex, and we are currently working on open-sourcing and sharing this part of the strategy (before the end of July). Please stay tuned for our future updates;)
